doc/API.md

```diff
@@ -15,6 +15,7 @@ Commands must be sent as valid JSONRPC 2.0 requests, ending with a `\n`.
 | [`delspendtx`](#delspendtx)                                 | Delete a stored Spend transaction                    |
 | [`broadcastspend`](#broadcastspend)                         | Finalize a stored Spend PSBT, and broadcast it       |
 | [`startrescan`](#startrescan)                               | Start rescanning the block chain from a given date   |
+| [`listtransactions`](#listtransactions)                     | List of transactions of incoming and outgoing funds  |
```

Comment:
The commit says " Add list_confirmed_transactions to Command " but this is listtransactions?